app/src/sharedTest/java/org/oppia/app/player/exploration/ExplorationActivityTest.kt

```diff
@@ -275,14 +278,14 @@ class ExplorationActivityTest {
         TEST_EXPLORATION_ID_2
       )
     ).use {
+      explorationDataController.startPlayingExploration(TEST_EXPLORATION_ID_2)
       onView(withId(R.id.action_audio_player)).check(matches(not(isDisplayed())))
     }
     explorationDataController.stopPlayingExploration()
   }
 
   @Test
   fun testAudioWithNoVoiceover_openPrototypeExploration_configurationChange_checkAudioButtonIsHidden() { // ktlint-disable max-line-length
```

Comment:
Try to remove `//ktlint` check everywhere.
One solution can be:
`testAudioWithNoVoiceover_openPrototypeExp_configChange_audioButtonNotVisible`

Reply:
It seems like a bit long issue to resolve and it's almost in all the files. I will create a good first issue for this.

Reply:
https://github.com/oppia/oppia-android/issues/1855